pycbc/waveform/ringdown.py

```diff
@@ -457,23 +457,28 @@ def fd_damped_sinusoid(f_0, tau, amp, phi, delta_f, f_lower, f_final, t_0=0.,
 #### Base multi-mode for all approximants
 ######################################################
 
-def multimode_base(input_params):
+def multimode_base(input_params, fd_approximant=False, td_approximant=False,
+                   freq_tau_approximant=False):
     """Return a superposition of damped sinusoids in either time or frequency
     domains with parameters set by input_params.
```

Comment:
Add a `Parameters` section to the doc string to explain what each keyword argument does. Also, you could combine this into a single, required argument `domain` that must be set to either `fd` or `td`. That'll simplify the logic below, and make it more straightforward to explain how to use the function.